logger.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+// ServiceLogger represents a logger with logging level prefixes for a specific service.
+type ServiceLogger struct {
+	ServiceName string
+	Debug       *log.Logger
+	Info        *log.Logger
+	Fatal       *log.Logger
+	Error       *log.Logger
+	*log.Logger
+}
+
+// NewServiceLogger returns a logger with designated logging levels for a particular service.
+func NewServiceLogger(serviceName string, debug bool) ServiceLogger {
+	logger := ServiceLogger{
+		serviceName,
+		log.New(os.Stdout, "DEBUG: "+serviceName+": ", log.LstdFlags|log.Lshortfile|log.Lmicroseconds),
+		log.New(os.Stdout, "INFO: "+serviceName+": ", log.LstdFlags|log.Lshortfile|log.Lmicroseconds),
+		log.New(os.Stdout, "FATAL: "+serviceName+": ", log.LstdFlags|log.Lshortfile|log.Lmicroseconds),
```

Comment:
So the Fatal logger is interesting to me because a call to it should always immediately be followed by a program exit. If an error is 'Fatal' it is an error that is an unrecoverable state--the service will (or really should) die... In that instance, `Fatal` feels like it should be abstracted to take a message, and an exit code and exit the process. This is not unlike how testings `.Fatalf` works. It stops execution of tests, closing the program.

Reply:
I think I have a preference here for being dumb, FATAL is meant to be additional logging to conveniently trigger our alarms and add additional context. I think panic will do as you described above (log, exit code, exit process). I imagine there could be possible situations where we want to indicate a severe problem has occurred but not panic and exit completely. However maybe within that context I'm using Error and Info logs improperly.

Reply:
In that case the `Fatal` logger should be removed entirely and `panic` used for fatal errors.

Reply:
Or optionally - use the word "Critical" for alarms instead of "Fatal" if you want an extra "this is a really bad error and you need to pay attention right now - sound the octo-alert!"